org.lflang/src/org/lflang/validation/AttributeSpec.java

```diff
@@ -210,9 +239,26 @@ enum AttrParamType {
         ));
         // @sparse
         ATTRIBUTE_SPECS_BY_NAME.put("sparse", new AttributeSpec(null));
+        // @language(lang)
+        ATTRIBUTE_SPECS_BY_NAME.put("language", new AttributeSpec(
+            List.of(new AttrParamSpec(AttributeSpec.VALUE_ATTR, AttrParamType.LANGUAGE, null))
+        ));
+        // @_fed_recv
+        ATTRIBUTE_SPECS_BY_NAME.put("_fed_recv", new AttributeSpec(null));
+        // @_fed_send
+        ATTRIBUTE_SPECS_BY_NAME.put("_fed_send", new AttributeSpec(null));
+        // @_fed_inp_ctrl
+        ATTRIBUTE_SPECS_BY_NAME.put("_fed_inp_ctrl", new AttributeSpec(null));
+        // @_fed_out_ctrl
+        ATTRIBUTE_SPECS_BY_NAME.put("_fed_out_ctrl", new AttributeSpec(null));
```

Comment:
The `_fed_recv`, `_fed_send`, `_fed_inp_ctrl` and `_fed_out_ctrl` are only used to mark reactions as unordered (see AttributeUtils). It would be cleaner to introduce an `@unordered` attribute instead.

Reply:
Fixed in https://github.com/lf-lang/lingua-franca/pull/1552